app/models/student_risk_level.rb

```diff
@@ -6,6 +6,11 @@ class StudentRiskLevel < ActiveRecord::Base
 
   # Use most recent assessments to calculate risk
   def mcas_math
+    # First look for the newer MCAS assessment
+    next_gen = student.latest_result_by_family_and_subject("Next Gen MCAS", "Mathematics")
+    return next_gen if next_gen
+
+    # If there are none, look for the original MCAS
```

Comment:
Another way to do this which might be cleaner: 

```
# student.rb

151: latest_result_by_family_and_subject(["Next Gen MCAS", "MCAS"], "Mathematics")
```

Reply:
Ah yes, forgot that it wouldn't care if you passed a single value or an array. Good call.